react/react1/week1/to-do/src/Components/ToDoList.jsx

```diff
@@ -0,0 +1,35 @@
+
+import ToDoItem from './ToDoItem'
+import ToDoList from './ToDoHeader'
+const listArray = [
+  {  task: "Get out of bed", deadLine: "Wed Sep 13 2017" },
+  { task: "Brush teeth", deadLine: "Thu Sep 14 2017" },
+  { task: "Eat breakfast", deadLine: "Fri Sep 15 2017" }
+];
+
+function ToDo() {
+  return (
+    <div className="header">
+      <ToDoList/>
+      <ItemKey/>
+      </div>  
+  );
+}
+function ItemKey(){
+return (
+    <ul className="list">
+{listArray.map((eachList,i)=> {
+ return <TaskItem taskItem={eachList} key={i}/>
+})}
+    </ul>
+)
+}
+function TaskItem({taskItem}){
```

Comment:
It's a good practice to separate function components in another file, or the code could get messy when it grows.

Reply:
I got it,actually I made already 3 files so thought to keep it here ;),thank you for suggestions